pkg/agent/storage/https.go

```diff
@@ -34,6 +34,7 @@ import (
 
 const (
 	HEADER_SUFFIX = "-headers"
+	DEFAULT_MAX_DECOMPRESSION_SIZE = 500 * 1024 * 1024 // 500MB
```

Comment:
@tessapham I think you need to rebase again, the PR includes other people's commits

Reply:
I made this change to fix linting errors.